ExposureNotification.App/ExposureNotification.App/Services/LocalState.cs

```diff
@@ -42,7 +42,9 @@ public static void Save()
 
 	public class LocalState
 	{
-		public static readonly Dictionary<string, ulong> DefaultServerBatchNumbers = new Dictionary<string, ulong> { { "ZA", 0 }, { "CA", 0 } };
+		public static Dictionary<string, ulong> DefaultServerBatchNumbers
```

Comment:
I would just remove this and use the settings and never store the regions in the state.